pages/common/aws-codecommit.md

```diff
@@ -0,0 +1,24 @@
+# aws codecommit
+
+> AWS CodeCommit is a secure, highly scalable, managed source control service that hosts private Git repositories.
+> More information: <https://aws.amazon.com/codecommit/>.
+
+- Install aws cli:
+
+`msiexec.exe /i https://awscli.amazonaws.com/AWSCLIV2.msi`
+
+- Run this command to verify that the CodeCommit commands for the AWS CLI are installed:
+
+`aws codecommit help`
+
+- Configure the AWS CLI with a profile:
+
+`aws configure`
+
+- To view a list of all available CodeCommit commands:
```

Comment:
```suggestion
- Display help and verify that the CodeCommit commands for the AWS CLI are installed:
```
I guess this would be better.
Or
```suggestion
- Display help:
```